python/ray/rllib/agents/agent.py

```diff
@@ -583,6 +583,19 @@ def session_creator():
             input_evaluation_method=config["input_evaluation"],
             output_creator=output_creator)
 
+    @override(Trainable)
+    def _export_default_policy(self, export_formats, export_dir):
+        exported = {}
+        if ExportFormat.CHECKPOINT in export_formats:
+            path = os.path.join(export_dir, ExportFormat.CHECKPOINT)
+            self.export_policy_checkpoint(path)
+            exported[ExportFormat.CHECKPOINT] = path
+        if ExportFormat.MODEL in export_formats:
+            path = os.path.join(export_dir, ExportFormat.MODEL)
+            self.export_policy_model(path)
+            exported[ExportFormat.MODEL] = path
+        return exported
```

Comment:
It would be nice to raise an error if the format isn't known

Reply:
this is taken care of in `validate`